eahub/feedback/migrations/0002_auto_20210117_1547.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.16 on 2021-01-17 15:47
```

Comment:
Ideally the 2 migration files would be one with the fields you've added now in the initinal set up.